### PR TITLE
Translate community into Japanese

### LINKED ISSUE #23123: Translate community pages into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [community](https://kubernetes.io/community) into Japanese.

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by shuuji3**
Closes #23123.

### CODE REVIEW

**Comment by shuuji3** [comment]
/assign @nasa9084

Page preview: https://deploy-preview-23125--kubernetes-io-master-staging.netlify.app/ja/community

File: content/ja/community/_index.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
+---
+title: コミュニティ
+layout: basic
+cid: community
+---
+
+<div class="newcommunitywrapper">
+<div class="banner1">
+  <img src="/images/community/kubernetes-community-final-02.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="desktop">
+  <img src="/images/community/kubernetes-community-02-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="mobile">
```
Comment (sftim) [inline content/ja/community/_index.html:10]: Optional: localize the `alt` attribute.

File: content/ja/community/_index.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+  </div>
+
+  <div class="imagecol">
+    <img src="/images/community/kubernetes-community-final-04.jpg" alt="Kubernetes Conference Gallery" style="width:100%" class="desktop">
+  </div>
+
+  <div class="imagecol" style="margin-right:0% important">
+    <img src="/images/community/kubernetes-community-final-05.jpg" alt="Kubernetes Conference Gallery" style="width:100%;margin-right:0% important" class="desktop">
+  </div>
+  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%;margin-bottom:3%" class="mobile">
```
Comment (sftim) [inline content/ja/community/_index.html:42]: Optional: localize the `alt` attribute.

File: content/ja/community/_index.html [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+<br class="mobile"><br class="mobile">
+<br class="tablet"><br class="tablet">
+<h1 style="padding-top:1%">ディスカッション</h1>
+
+<div style="font-weight:300;text-align:center">さまざまな話題について話し合いが行われています。次のようなプラットフォームで会話に参加してください。</div>
+
+<div class="resourcecontainer">
+
+<div class="resourcebox">
+<img src="/images/community/discuss.png" alt=Forum" style="width:80%;padding-bottom:2%">
```
Comment (sftim) [inline content/ja/community/_index.html:123]: Optional: localize the `alt` attribute.

File: content/ja/community/code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+layout: basic
+cid: community
+css: /css/community.css
+---
+
+<div class="community_main">
+<h1>Kubernetesコミュニティ行動規範</h1>
+
+Kubernetesは、<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/jp.md">CNCF行動規範</a>に従います。
+以下のCNCFの行動規範のテキストは、<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit da91d9e</a>から複製したものです。この掲示が古くなっていることに気がついた場合には、<a href="https://github.com/kubernetes/website/issues/new">issueを作成</a>してください。
```
Comment (sftim) [inline content/ja/community/code-of-conduct.md:12]: nit: https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct-languages/jp.md

File: content/ja/community/_index.html [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+-->
+
+<div class="news">
+<br class="mobile"><br class="mobile">
+<br class="tablet"><br class="tablet">
+<h1 style="margin-bottom:2%">最新ニュース</h1>
+
+<br>
+<div class="twittercol1">
+<a class="twitter-timeline" data-tweet-limit="1" href="https://twitter.com/kubernetesio?ref_src=twsrc%5Etfw">Tweets by kubernetesio</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (sftim) [inline content/ja/community/_index.html:229]: I would translate at least “by”, eg “@kubernetesioのtweet”

File: content/ja/community/code-of-conduct.md (hunk 4 above)
Comment (sftim) [reply content/ja/community/code-of-conduct.md:12]: Also `da91d9e` != `214585e` (is that wrong upstream too?)

File: content/ja/community/_index.html (hunk 1 above)
Comment (shuuji3) [reply content/ja/community/_index.html:10]: Right, I'd better translate the `alt` attribute for better accessibility.

File: content/ja/community/_index.html (hunk 5 above)
Comment (shuuji3) [reply content/ja/community/_index.html:229]: Nice! "@kubernetesioのtweet" is correct. Additionally, I'd like to translate to "@kubernetesさんのツイート" following the expression on Twitter.

File: content/ja/community/code-of-conduct.md (hunk 4 above)
Comment (shuuji3) [reply content/ja/community/code-of-conduct.md:12]: Yes, `214585e` is the latest one. I took `da91d9e` from the commit history of Japanese translation but it was a little old.

File: content/ja/community/_index.html (hunk 5 above)
Comment (inductor) [reply content/ja/community/_index.html:229]: @shuuji3 Please don't be cause https://twitter.com/kubernetes is owned by a random person lol

Comment (shuuji3) [reply content/ja/community/_index.html:229]: Oh, that's right!😂

**Pull request merged**